sound/soc/codecs/rt1308-sdw.c

```diff
@@ -131,7 +131,7 @@ static int rt1308_read_prop(struct sdw_slave *slave)
 
 	/* for sink */
 	nval = hweight32(prop->sink_ports);
-	num_of_ports += nval;
+	num_of_ports = nval;
```

Comment:
sorry, I don't understand how and where this patch fixes any "off by 1" error? `x * 1 == x` in "normal" arithmetics. I agree, that this simplifies the code, but The title of the patch seems wrong?

Reply:
see line 121, the init was 1.

Reply:
@plbossart maybe indeed I need glasses... See my comment too - I thought it was a multiplication, not an addition... Black on red doesn't read too well here :-/